health/notifications/alerta/README.md

```diff
@@ -84,11 +72,11 @@ We can test alarms using the standard approach:
 /opt/netdata/netdata-plugins/plugins.d/alarm-notify.sh test
 ```
 
-Note: Netdata will send 3 alarms, and because last alarm is "CLEAR"
-you will not see them in main Alerta page, you need to select to see
-"closed" alarm in top-right lookup. A little change in `alarm-notify.sh`
-that let us test each state one by one will be useful.
+> **Note** This script will send 3 alarms. 
+> Alerta will not show the alerts in the main page, because last alarm is "CLEAR".
+> To see the test alarms, you need to select "closed" alarms in the top-right lookup. 
+>A little change in `alarm-notify.sh` that let us test each state one by one will be useful.
```

Comment:
I tried to find the author but it seems it was Costa who added this (the [initial contribution](https://github.com/netdata/netdata/pull/3160) has not readme).

I think we can just remove this line (anyway, advising to slightly change the `alarm-notify.sh` script in order to test something doesn't sound like a good idea)